changelogs/unreleased/ensure-graceful-shutdown-inmanta-container.yml

```diff
@@ -0,0 +1,4 @@
+---
+description: Ensure that the inmanta server running in the container gracefully shuts down on termination
+change-type: patch
+destination-branches: [master]
```

Comment:
I think merge-tool still require the three branches here no? Even for the open source product